codecov.yml

```diff
@@ -6,6 +6,13 @@ comment:
 github_checks:
   annotations: false
 
+coverage:
+  status:
+    patch:
+      default:
+        target: auto
+        threshold: 33%
+
```

Comment:
The checker suggests to me that my PR is reduced by more than 10% coverage/patch, I'm not quite sure what this coverage/patch means. But I think this is not a reasonable request, I should at most only need to provide the test code changes related to my change file, other tests of unrelated code should not be written by the PR submitter.

Reply:
Maybe we can consider turning it off and just keep the codecov/project detection. For the vast majority of projects, the first thing that needs our attention is whether all the features that users need, are implemented and whether the number of bugs is less. Rather than a cold technical indicators. I also found an interesting situation when I was querying the configuration parameters of codecov/patch, there were many people searching how to lower the indicator or turn it off.